app/views/posts/index.html.erb

```diff
@@ -0,0 +1,37 @@
+
+<div class="user-post">
+
+    <div class="user-card">
+          <div class="user-image">
+              <img src="<%= @user.photo_link %>" alt="<%= @user.name %> photo" class="img-fluid rounded-circle">
+          </div>
+        <div class="user-info">
+          <h3><%= @user.name %></h3>
+          <p class="comment">Number of posts: <%= @user.posts.count %></p>
+       </div>
+    </div>
+      <% @posts.each_with_index do |post, index| %>
+      <div class="allposts-user">
+      
+       <h4><%= link_to "Post #{index + 1}", user_post_path(@user, post) %> </h4>
+      <p><%= post.text %></p>
+      <div class="likes">
+         <p class="comment">comments: <%= post.comments.count %></p>
+        <p class="likes"> likes: <%= post.likes.count %></p>
+      </div>
+    </div>  
+      <% end %>
+  <div class="all-comments">
+  <% @posts.each do |post| %>
+    <ul>
+      <% post.comments.each do |comment| %>
+        <li>
+          <strong><%= comment.user.name %></strong>: <%= comment.text %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+</div>
+<div class="pagination">
+    <%= will_paginate @posts, :page_links => false %>
+</div>
```

Comment:
- [ ] Kindly pay attention to the detail and place the elements where they belong per the wireframe.

  Your Implementation | Wireframe layout
  :---:|:---:
  ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/decf2e3c-0bd9-4171-a0af-72112408cc4c) | ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/016f1697-2efa-43dd-8a20-1cd0af0cb463)